queries.sql

```diff
@@ -5,4 +5,57 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon','Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg>10.5;
 SELECT * FROM animals WHERE neutered='True';
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* Day 2 */
+
+BEGIN;
+
+UPDATE animals SET species='unspecified';
+
+SELECT species FROM animals;
+ROLLBACK;
+
+SELECT species FROM animals;
+
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE name NOT LIKE '%mon';
+
+COMMIT;
+
```

Comment:
- [ ] This transaction has not been **BEGIN** 😨 Kindly keep this in mind that when you `COMMIT` or completely `ROLLBACK` (means without Savepoint) then it means that your transaction has been completed than you will have to create next transaction using `BEGIN` command but i don't see any `BEGIN` command here 😨 Kindly add it 👍